operator/cmd/ccnp_event.go

```diff
@@ -128,6 +128,7 @@ func enableCCNPWatcher(ctx context.Context, wg *sync.WaitGroup, clientset k8sCli
 
 	mgr.UpdateController("ccnp-to-groups",
 		controller.ControllerParams{
+			Group: "ccnp-to-groups",
```

Comment:
💡 idea: since we're not bound by the controller identifier, maybe we could make these more readable: ```ciliumclusterwidenetworkpolicy-to-groups```

(might actually be too hard to type in this situation)

Reply:
I like the idea of extra verbosity, I'll consider this when moving these names into constants.